### PR TITLE
replace empty ESX compatibility guide (noref)

### PR DESCRIPTION
**Pull request by csymons-suse**
the link in ESX Guest OS Support went to an empty form.
Replace with form filled in with proper information.

Remove FIXME:

(cherry picked from commit 877b2d8fcf2434179c9ffa610937a5c6ce68583b)

### CODE REVIEW

File: xml/planning-planning-hw_support_esxguestos.xml [hunk 1]
```diff
@@ -4,11 +4,10 @@
...
  <title>ESX Guest OS Support</title>
- <!-- FIXME: This link leads to an empty search form instead of to a document
- - sknorr, 2018-03-02 -->
  <para>
-  For ESX, refer to the
-  <link xlink:href="http://www.vmware.com/resources/compatibility/search.php?action=search&amp;deviceCategory=software&amp;advancedORbasic=advanced&amp;maxDisplayRows=50&amp;key=&amp;productId=4&amp;gos_vmw_product_release%5B%5D=90&amp;datePosted=-1&amp;partnerId%5B%5D=-1&amp;os_bits=-1&amp;os_use%5B%5D=-1&amp;os_family%5B%5D=-1&amp;os_type%5B%5D=-1&amp;rorre=0">&vmware;
-  Compatibility Guide</link>.
+  For ESX, refer to the <link
+  xlink:href="https://www.vmware.com/resources/compatibility/search.php?deviceCategory=software&amp;details=1&amp;releases=273,274,338&amp;productNames=15&amp;page=1&amp;display_interval=500&amp;sortColumn=Partner&amp;sortOrder=Asc&amp;testConfig=16">&vmware;
+  Compatibility Guide - scroll down</link>.
```
Comment (asettle) [inline xml/planning-planning-hw_support_esxguestos.xml]: I'm not sure I understand the "scroll down" part? That's not part of the title page.

Comment (csymons-suse) [reply xml/planning-planning-hw_support_esxguestos.xml]: The link goes to a page with a form at the top. The compatibility information is not visible even though the form has been completed with the appropriate information. The proper compatibility information is visible with "scroll down". Suggestions?

Comment (asettle) [reply xml/planning-planning-hw_support_esxguestos.xml]: Thought that might be the case, bummer. I did look at the page and wasn't too sure on suggestions, but wanted to check that's what you intended.
So, maybe...

> For ESX, refer to the <link xlink:href="https://www.vmware.com/resources/compatibility/search.php?deviceCategory=software&amp;details=1&amp;releases=273,274,338&amp;productNames=15&amp;page=1&amp;display_interval=500&amp;sortColumn=Partner&amp;sortOrder=Asc&amp;testConfig=16">&vmware; Compatibility Guide</link> for more information. When you access this link, the information you require is at the bottom of the page.

Or something similar? Just an instruction detailing the action.

**Comment by csymons-suse** [comment]
link reworked in https://github.com/SUSE-Cloud/doc-cloud/pull/1052
https://github.com/SUSE-Cloud/doc-cloud/commit/1a222b24

**Pull request merged**